pldoc/static/js/ui.js

```diff
@@ -20,6 +20,12 @@ define([
 
         setupHtml: function() {
             $('html').removeClass('no-js');
+            $("#pldoc-nav-elements ul li a").each(function (index, element) {
+                if ( $( element ).attr( "href" ) === window.location.pathname ) {
+                  $( element ).addClass("is-current");
+                  return false;
```

Comment:
That breaks out of the each() when we find the href that matches
window.location.pathname, preventing unnecessary iterations.
On Thu, Oct 15, 2015 at 15:36 AlasdairSwan <notifications@github.com> wrote:

> In pldoc/static/js/ui.js
> <https://github.com/edx/ux-pattern-library/pull/197#discussion_r42169849>:
>
> > @@ -20,6 +20,12 @@ define([
> >
> >          setupHtml: function() {
> >              $('html').removeClass('no-js');
> > +            $("#pldoc-nav-elements ul li a").each(function (index, element) {
> > +                if ( $( element ).attr( "href" ) === window.location.pathname ) {
> > +                  $( element ).addClass("is-current");
> > +                  return false;
>
> Why return false?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/ux-pattern-library/pull/197/files#r42169849>.
>